README.de.md

```diff
@@ -0,0 +1,97 @@
+# Dlool (Deutsch)
+
+## Ein digitales und kollaboratives Hausaufgabenheft
+
+Dlool ist ein digitales und kollaboratives Hausaufgabenheft.  
+Klassische Hausaufgabenhefte haben eine Menge Probleme. Hier sind einige Probleme und wie Dlool Du löst:
+
+| Problem                                                                                                  | Lösung                                                                                                                                                                                                                                                                                                 |
+| -------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Manche Schüler arbeiten nur digital, benutzen aber immer noch ein klassisches, analoges Hausaufgabenheft | Dlool ist digital und ein optimaler Ersatz.                                                                                                                                                                                                                                                            |
+| Alle in einer Klasse schreiben die selben Hausaufgaben auf                                               | Dlool ist kollaborativ, alle Einträge einer Klasse sind für alle verfügbar.                                                                                                                                                                                                                            |
+| Du sind nicht sehr gut für die Umwelt, da Du nach nur einem Schuljahr weggeworfen werden               | Dlool ist digital, es wird kein Papier benötigt.                                                                                                                                                                                                                                                       |
```

Comment:
You can't just search and replace *Sie* with *Du* it has way to many different usecases. This problem is in multiple rows of this table.